cli/errors.py

```diff
@@ -0,0 +1,12 @@
+"""
+Provide errors for command line interface.
+"""
+
+
+class NotSupportedOperatingSystemToGetNodePrivateKeyError(Exception):
```

Comment:
Maybe `NotSupportedOSToGetNodePrivateKeyError` is better, what you think @dmytrostriletskyi ?

Reply:
@Alladin9393, yes, a bit shorter.